src/configs/starship_root.rs

```diff
@@ -119,6 +119,7 @@ pub const PROMPT_ORDER: &[&str] = &[
     "os",
     "shell",
     "character",
+    "direnv",
```

Comment:
Is there a particular reason, why the `[direnv]` module is shown after the `[character]` module? I would be confused, if the `[character]` module wouldn't be the last?

Reply:
Nope,just a hasty addition. Moved it before `env_var` since there's a natural relationship between the two modules.